dev/com.ibm.ws.container.service/src/com/ibm/ws/container/service/annocache/internal/AnnotationsAdapterImpl.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018, 2023 IBM Corporation and others.
+ * Copyright (c) 2018 IBM Corporation and others.
```

Comment:
```suggestion
 * Copyright (c) 2018, 2023 IBM Corporation and others.
```

Again I don't know why removing the second date from the copyright.